en/guide/mavlink_2.md

```diff
@@ -1,28 +1,72 @@
 # MAVLink 2
 
-MAVLink version 2 is a backward-compatible new version. The main new features are:
+*MAVLink 2* is a backward-compatible update to the MAVLink protocol that has been designed to bring more flexibility and security to MAVLink communication.
 
-* Support for more than 256 message IDs \(now 24 bit or over 16 million packets\)
-* Packet signing \(authentication\)
-* Support for extending existing messages
-* Support for variable length arrays
+The key new features of *MAVLink 2* are:
+* More than 256 message IDs (24 bit message ID - over 16 million packets)
+* [Packet signing](../guide/message_signing.md) (authentication)
+* [Extending existing MAVLink messages](#message_extensions)
+* [Variable length arrays](#variable_arrays) (*simplified implementation*).
 
+*MAVLink 2* bindings have been developed for C and Python (other languages are not implemented yet).
 
+> **Tip** We recommend you start by reading the *MAVLink 2* [design document](https://docs.google.com/document/d/1XtbD0ORNkhZ8eKrsbSIZNLyg9sFRXMXbsR2mp37KbIg/edit?usp=sharing)
 
-## Upgrading an existing C installation
 
-The existing [v1 library ](https://github.com/mavlink/c_library_v1)can be simply upgraded by dropping in the v2 library from Github:
+## Upgrading an Existing C Installation
 
-[https://github.com/mavlink/c\_library\_v2](https://github.com/mavlink/c_library_v2)
+The existing *MAVLink 1* pre-built library [mavlink/c_library_v1](https://github.com/mavlink/c_library_v1) can be upgraded by simply dropping in the *MAVLink 2* library from Github: [mavlink/c_library_v2](https://github.com/mavlink/c_library_v2)
 
-The v2 library will send packets in MAVLink v2 framing by default. In order to default to v1, run this code snippet on boot:
+*MAVLink 2* usage is then covered in the following section.
+
+
+## Using the C Implementation
+
+For most users usage of the pre-generated C headers is recommended: https://github.com/mavlink/c_library_v2
+
+This will generate a set of C headers in the generator/C/include_v2.0 directory. 
+These headers offer the same range of APIs as was offered by MAVLink1. 
+
+The major changes from an API perspective are:
+
+* you don't need to provide a message CRC table any more, or message length table. 
```

Comment:
Is this relevant? Was this created for people, or something they had to do manually?